pom.xml

```diff
@@ -497,6 +497,32 @@
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-checkstyle-plugin</artifactId>
                     <version>3.3.1</version>
+                    <configuration>
+                        <violationSeverity>warning</violationSeverity>
+                        <failOnViolation>true</failOnViolation>
+                        <failsOnError>true</failsOnError>
+                        <logViolationCountToConsole>true</logViolationCountToConsole>
+                        <logViolationsToConsole>true</logViolationsToConsole>
+                        <includeTestSourceDirectory>true</includeTestSourceDirectory>
+                    </configuration>
+                    <executions>
+                        <execution>
+                            <id>layering</id>
+                            <phase>verify</phase>
+                            <goals>
+                                <goal>check</goal>
+                            </goals>
+                            <configuration combine.children="override">
```

Comment:
Would we be better off with a ` <skip>${skip.layering.checkstyle}</skip>` property set to false in the base pom and an override in the other pom. The combine.children style is a bit funky.

Reply:
@robobario, @k-wall was suggesting [this change](https://github.com/tombentley/kproxy/pull/16/files), which I assume would work for you?

Reply:
yep that works